code-ready/crc
# Issue #1286 Add log on as user permission for current user during setup

Fixes #1286 

~in order to use the LocalSystem account to run the daemon service
we also need to take care of where the .crc dir is created, as by
default when using the LocalSystem account the users home dir is at
C:\Windows\System32\config\systemprofile but we want the .crc dir to
be created in the logged in user's home directory.~

~We capture the user's home dir when crc setup is run and pass it as an
argument in the service invocation command, e.g `crc.exe daemon c:\Users\anjan`~

Dropped the commit using the `LocalSystem` account, instead we use the current user account to the run the daemon service, and also allow _log on as a service_ for the user, so that a crc user doesn't have to do it manually.

### How to test:
1. run `crc setup --enable-experimental-features --log-level debug`
2. Should have a working tray up and running
3. If errors are encountered then Open and check if `secpol.msc` > Local User Rights > Log on as service contains the current user's name.
